src/stories/LossSelection.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { ComponentMeta } from '@storybook/react';
+import HistorySelection, { HistorySelectionProps } from './HsitorySelect';
+
+export default {
+    component: HistorySelection,
+    title: 'History Selection/HistorySelection',
+} as ComponentMeta<typeof HistorySelection>;
+
+function Template(args: HistorySelectionProps) {

```

Comment:
According a simple example here with React TS, we can remove the imports of types:
https://storybook.js.org/docs/react/get-started/whats-a-story

And use ComponentStory for the common template as well: 
```const Template: ComponentStory<typeof HistorySelection> = (args) => <HistorySelection {...args} />;```